damus/Views/ParicipantsView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  ParicipantsView.swift
+//  damus
+//
+//  Created by Joel Klabo on 1/18/23.
+//
+
+import SwiftUI
+
+struct ParticipantsView: View {
+    
+    let damus_state: DamusState
+    
+    @Binding var references: [ReferencedId]
+    @Binding var originalReferences: [ReferencedId]
+    
+    var body: some View {
+        VStack {
+            Text("Edit participants")
```

Comment:
@joelklabo Since you're a core contributor and you might be submitting changes with strings frequently, just wanted to note for next time that you should add comments to localized strings so that the translators know how to approach translations. No big deal, I'm going to add comments in a separate PR. And I'm going to see if I can add a CI step to catch strings with no comments to fail the build.

Reply:
Yeah, I realized that after it got merged. Thanks!